plugins/woocommerce/changelog/restrict_paypal_standard

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: update
+
+Only load the PayPal Standard payment method on stores that have connected their account or have existing PayPal orders.
```

Comment:
Beware, there are two changelog files.

Reply:
Hey @Konamiman, just checking. Is two changelog files okay?
I was thinking one changelog for deprecating the filter and another for the changes to the default `_should_load` and upgrade script.

Reply:
Yes, I asked just to be sure and looks like it's ok to have more than one changelog file in the same pull request.